docs/src/CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+
+# Change Log
+All notable changes to this project will be documented in this file.
+ 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+ 
+## [0.2] - yyyy-mm-dd
```

Comment:
assume this yyyy-mm-dd is a placeholder for the day we actually release the module?

Reply:
Yeah, I figured that would be best to set durning the actual release.